metakube/resource_cluster_schema.go

```diff
@@ -147,7 +147,7 @@ func metakubeResourceClusterSpecFields() map[string]*schema.Schema {
 			DiffSuppressFunc: func(_, _, _ string, d *schema.ResourceData) bool {
 				configured, ok := d.GetOkConfigured("spec.0.cni_plugin.0.type")
 				newValue := d.Get("spec.0.cni_plugin.0.type")
-				return (!ok || configured == "") && newValue == "canal"
+				return (!ok || configured == "") && (newValue == "canal" || newValue == "cilium")
```

Comment:
what would happen if a cluster was created while default was canal, and we want to switch it to cilium on purpose?